### PR TITLE
Gea feature/model change

### PR DESCRIPTION
**Pull request by Ana00000**

### CODE REVIEW

**Comment by Ana00000** [comment]
[(5) Upgrade of a database](https://trello.com/c/hPKa8g2a/71-5-upgrade-of-a-database)

**Comment by Ana00000** [comment]
[(1) GEA room removed](https://trello.com/c/wovo1xhQ/73-1-gea-room-removed)

File: GraphicEditor/Repositories/EquipmentRepository.cs [hunk 1]
```diff
@@ -3,22 +3,21 @@
...
         private MySqlConnection connection;
+
         public EquipmentRepository()
         {
             connection = new MySqlConnection("server=localhost;port=3306;database=mydb;user=root;password=root");
         }
 
         private List<Equipment> GetEquipments(String query)
         {
+            connection.Close();
```
Comment (milicasiriski) [inline GraphicEditor/Repositories/EquipmentRepository.cs:20]: Why?

File: GraphicEditor/Repositories/FloorRepository.cs [hunk 2]
```diff
@@ -9,13 +9,16 @@ namespace GraphicEditor.Repositories
...
+        private RoomRepository roomRepository = new RoomRepository();
+
         public FloorRepository()
         {
             connection = new MySqlConnection("server=localhost;port=3306;database=mydb;user=root;password=root");
         }
 
         private List<Floor> GetFloors(String query)
         {
+            connection.Close();
```
Comment (milicasiriski) [inline GraphicEditor/Repositories/FloorRepository.cs:21]: 👀

File: GraphicEditor/Repositories/MedicineManufacturerRepository.cs [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    {
+        private MySqlConnection connection;
+        public MedicineManufacturerRepository()
+        {
+            connection = new MySqlConnection("server=localhost;port=3306;database=mydb;user=root;password=root");
+        }
+
+        private List<MedicineManufacturer> GetMedicineManufacturers(String query)
+        {
+            connection.Close();
```
Comment (milicasiriski) [inline GraphicEditor/Repositories/MedicineManufacturerRepository.cs:19]: Why is this everywhere?

File: GraphicEditor/View/Windows/EquipmentSearch.xaml.cs [hunk 4]
```diff
@@ -27,47 +26,45 @@ private void SearchEquipmentOrMedicine(object sender, RoutedEventArgs e)
...
             if (medicines == null)
             {
-                medicines = new List<MedicineGEA>();
+                medicines = new List<Medicine>();
             }
             if (equipments.Count != 0)
                 SearchEquipmentTextBlock.Text = ReportOnFoundEqipment(itemName, equipments);
             else if (medicines.Count != 0)
-                SearchEquipmentTextBlock.Text = ReportOnFoundMedicine(itemName, medicines);
+                SearchEquipmentTextBlock.Text = "";//= ReportOnFoundMedicine(itemName, medicines);
```
Comment (milicasiriski) [inline GraphicEditor/View/Windows/EquipmentSearch.xaml.cs]: If this is some bug or temporary fix put a TODO comment

File: GraphicEditor/View/Windows/EquipmentSearch.xaml.cs [hunk 5]
```diff
@@ -82,9 +79,9 @@ private string ReportOnFoundEqipment(string equipmentName, List<Equipment> equip
                 int checkCounter = 0;
                 foreach (Equipment equipment in equipments)
                 {
-                    List<RoomGEA> rooms = roomRepository.GetRoomsBySerialNumber(equipment.RoomId);
+                    Room room = roomRepository.GetRoomBySerialNumber(equipment.RoomId);
                     resultOfSearch += "\nInformation about rooms: ";
-                    resultOfSearch += RoomSearch.ReportOnFoundRooms(equipment.RoomId, rooms);
+                    //resultOfSearch += RoomSearch.ReportOnFoundRooms(equipment.RoomId, room);
```
Comment (milicasiriski) [inline GraphicEditor/View/Windows/EquipmentSearch.xaml.cs]: same here

**Review by milicasiriski — CHANGES_REQUESTED** [review]
What is going on with db connections?

File: GraphicEditor/Repositories/MedicineManufacturerRepository.cs (hunk 3 above)
Comment (Ana00000) [reply GraphicEditor/Repositories/MedicineManufacturerRepository.cs:19]: We are calling other repositories in the time of the transactions.

**Review by milicasiriski — APPROVED** [review]

**Review by VladimirBudjen — APPROVED** [review]

**Pull request merged**